docs/admin/cold-starts.md

```diff
@@ -0,0 +1,14 @@
+# Improve cold starts 
+
+Cloud Run comes with a number of settings that can improve performance around cold starts. These
+aren't set by default, as these settings can increase the running costs of the service. 
+
+To prevent cold starts, Cloud Run supports setting [minimum instances](https://cloud.google.com/run/docs/configuring/min-instances). If the minimum instance is set to "1", Cloud Run will always be running, and never start cold, thus eliminating this performance issue. Check the [Cloud Run pricing](https://cloud.google.com/run/pricing) before configuring this setting.  
```

Comment:
s/Cloud Run will always be running/a container instance will always be running/.

Reply:
might want to say something about the tradeoff in the cost between having something always on and unused and latency of starting the container and lost customers or something.